certification/internal/policy/container/base_on_ubi_test.go

```diff
@@ -59,45 +60,21 @@ var _ = Describe("BaseOnUBI", func() {
 		imageRef        certification.ImageReference
 	)
 
-	const osrelease = "os-release"
-
 	BeforeEach(func() {
 		fakeImage := fakecranev1.FakeImage{
```

Comment:
You can just create the FakeImage without any params.

Reply:
it didnt like it .. was getting null pointer so put the config back but will see what exactly was missing..

Reply:
just passing an empty config now